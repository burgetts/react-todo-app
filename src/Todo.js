import React from 'react';

const Todo = ({id, text, deleteTodo}) => {
    return (
        <>
            <li> {text} </li>
            <button onClick={() => deleteTodo(id)}> ‚ùå </button>
        </>
    )
}

export default Todo;