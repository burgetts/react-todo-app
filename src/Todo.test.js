import React from 'react';
import {render, fireEvent} from '@testing-library/react'
import Todo from './Todo'

it('renders without crashing', function () {
    render(<Todo text="test this component" />)
})

it('matches snapshot', function () {
    const {asFragment} = render(<Todo text="test this component" />)
    expect(asFragment()).toMatchSnapshot()
})