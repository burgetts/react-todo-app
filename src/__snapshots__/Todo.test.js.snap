// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li>
     test this component 
  </li>
  <button>
     ‚ùå 
  </button>
</DocumentFragment>
`;
